bezier/src/lib.rs

```diff
@@ -249,6 +260,32 @@ impl QuadraticBezierSegment {
     }
 }
 
+impl HasBoundingRect for QuadraticBezierSegment {
+    fn bounding_rect(&self) -> Rect {
+        let min_x = self.from.x.min(self.ctrl.x).min(self.to.x);
+        let max_x = self.from.x.max(self.ctrl.x).max(self.to.x);
+        let min_y = self.from.y.min(self.ctrl.y).min(self.to.y);
+        let max_y = self.from.y.max(self.ctrl.y).max(self.to.y);
+
+        return Rect::new(Point::new(min_x, min_y), Size::new(max_x - min_x, max_y - min_y));
```

Comment:
It looks like the euclid doc you are looking at is out of date, [the one in docs.rs](https://docs.rs/euclid/0.13.0/euclid/rect/fn.rect.html) has what we are looking for.